components/CardHeader.vue

```diff
@@ -3,13 +3,17 @@
     <img class="logo" src="/logo--unocha.svg" alt="Office for the Coordination of Humanitarian Affairs">
     <div class="meta">
       <h1 class="title">{{ title }}</h1>
```

Comment:
I'm not sure where this comes from. Is it already translated? (I will know more when I can see the data locally)

Reply:
Aha moment. Now that I've looked in contentful, I can see there's a title field per sit rep. I assume we don't worry about translating contentful data, or this will be done in another manner... Using https://reports.unocha.org/country/burundi/ as example, I can see the content was entered in French and does not change when the UI language does.

Reply:
@left23 the source of `title` is obfuscated due to `CarHeader` and `CardFooter` being sub-components of `Card` (unofficial term which reflects the implementation). Normally you'd look in `props`/`data`/`computed` for this value.

While fiddling with this originally I ended up stuffing the current SitRep's title in the Vuex Store (`this.$store...`), in order to expose as a global-ish var accessible from any component. Maybe it's smart and maybe I missed out on some inheritance thing for components-in-components.